acceptance/config/windows-aio-2012-x86_64.cfg

```diff
@@ -0,0 +1,32 @@
+HOSTS:
+  ubuntu1204:
+    roles:
+      - master
+      - database
+      - dashboard
+    platform: ubuntu-12.04-amd64
+    template: ubuntu-1204-x86_64
+    hypervisor: vcloud
+  w2012-puppetx64:
+    ruby_arch: x64
+    roles:
+      - agent
+      - default
+    platform: windows-2012-x86_64
+    template: win-2012-x86_64
+    hypervisor: vcloud
+  w2012-puppetx86:
+    ruby_arch: x86
```

Comment:
PR opened at https://github.com/puppetlabs/beaker/pull/769 to discuss ... might end up supporting both `:ruby_arch` (to fall inline with what Puppet has been doing) and `:install_32`